phpBB/language/en/acp/board.php

```diff
@@ -52,6 +52,8 @@
 	'DISABLE_BOARD_EXPLAIN'			=> 'This will make the board unavailable to users who are neither administrators nor moderators. You can also enter a short (255 character) message to display if you wish.',
 	'DISPLAY_LAST_SUBJECT'			=> 'Display subject of last added post on forum list',
 	'DISPLAY_LAST_SUBJECT_EXPLAIN'	=> 'The subject of the last added post will be displayed in the forum list with a hyperlink to the post. Subjects from password protected forums and forums in which user doesn’t have read access are not shown.',
+	'DISPLAY_UNAPPROVED_POSTS'		=> 'Display unapproved posts to the author',
+	'DISPLAY_UNAPPROVED_POSTS_EXPLAIN'	=> 'Unapproved posts in the moderation queue can be viewed by the author. Does not apply to Guest posts',
```

Comment:
That sounds for me rasther like unnaproved post's author has access to the moderartion queue in MCP, where that author can view tha unapprovad post.
What is the real behavior? I guess unapproved post can be viewed by its author only in vewtopic, but not in the moderation queue.
Probably it's worth rephrasing?

Reply:
`Unapproved posts still to be moderated can be viewed by its author. Does not apply to Guest posts`

Reply:
I propose changing the text to: 'Unapproved posts can be viewed by the author while waiting to be moderated. Does not apply to Guest posts'

Reply:
`Users (excepting guests) will be able to see their posts waiting for approval while vieving topics`, or something?
BTW, will the user also be able to quote/edit/delete their unapproved posts?

Reply:
As it stands, an author (with normal privileges) can see the post, can report it, and can edit it, but not quote or delete it.

Reply:
I'd recommend changing it to: `Unapproved posts can be viewed by their author. Does not apply to Guest posts.`